### PR TITLE
added css position: relative

### LINKED ISSUE #5444: [Term Entry] CSS: Position `relative`
### Reason/inspiration (optional)

We would like a new entry under CSS for the `relative` term. The entry should go in a new file under `content/css/concepts/position/terms/relative/relative.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use
- An image or GIF showcasing this position type can be a plus point.

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ozearkhan**
---

### Description

This PR adds a new term entry for "Relative Position" in CSS. The entry is located under `content/css/concepts/position/terms/relative/relative.md`. It explains what relative positioning is, how it works, and includes both the syntax and practical examples to show how you can use it in your CSS.

The first example is a simple demo of how to move elements around using relative positioning. The second example gets a little more advanced, covering things like z-index and percentage-based positioning. I’ve also added two images in the` media/ `folder to give a visual representation of what the examples look like when rendered.

### Issue Solved

Closes #5444

### Type of Change

- New Term for Existing Concept Entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

---

### CODE REVIEW

File: content/css/concepts/position/terms/relative/relative.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - 'Web Design'
+  - 'Web Development'
+  - 'CSS'
+Tags:
+  - 'positioning'
+  - 'CSS'
+  
+CatalogContent:
+  - 'learn-css'
+  - 'paths/css'
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:14]:
```suggestion
---
Title: 'relative' 
Description: 'Positions an element relative to its normal document flow placement, allowing for improved styling control.' 
Subjects: 
  - 'Web Design'
  - 'Web Development'
Tags:
  - 'CSS'
  - 'Positioning'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
```

Comment (mamtawardhani) [reply content/css/concepts/position/terms/relative/relative.md:14]: Modifications: Title should be the term, subjects should be picked from the [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) file, tags should be in alphabetical order.
Rephrased the description for better readability

File: content/css/concepts/position/terms/relative/relative.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+---
+
+The **relative** value of the `position` property allows an element to be positioned relative to its normal position in the document flow. This positioning method provides a way to adjust an element's position without affecting the layout of surrounding elements.
```
[the remark this answers is not in the document]
Comment (mamtawardhani) [reply content/css/concepts/position/terms/relative/relative.md:17]: relative should be in backticks as well as bold

File: content/css/concepts/position/terms/relative/relative.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## Syntax
+```css
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:19]:
```suggestion
```pseudo
```

Comment (mamtawardhani) [reply content/css/concepts/position/terms/relative/relative.md:19]: syntax should be in a ```pseudo``` block

File: content/css/concepts/position/terms/relative/relative.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+position: relative;
+```
+When an element is set to `position: relative`, you can use the `top`, `right`, `bottom`, and `left` properties to offset it from its normal position:
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:22]:
```suggestion
When an element is set to `position: relative`, the `top`, `right`, `bottom`, and `left` properties can be used to offset it from its normal position as follows:
```

Comment (mamtawardhani) [reply content/css/concepts/position/terms/relative/relative.md:22]: pronouns like "you" and "we" are not used in docs

File: content/css/concepts/position/terms/relative/relative.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  top: <length>;
+  right: <length>;
+  bottom: <length>;
+  left: <length>;
+}
+```
+
+- `<length>`: Can be specified in pixels (px), ems (em), rems (rem), percentages (%), or other CSS units.
+- Positive values move the element away from the specified edge.
+- Negative values move the element towards the specified edge.
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:36]:
```suggestion
- `<length>`: Can be specified in pixels (px), ems (em), rems (rem), percentages (%), or other CSS units. Positive values move the element away from the specified edge and negative values move it towards the specified edge.
```

File: content/css/concepts/position/terms/relative/relative.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+## Example 1:Simple Relative Positioning
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:38]:
```suggestion
## Example 1
```

File: content/css/concepts/position/terms/relative/relative.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+```
+
+Here's what the above examples look like:
+
+![CSS Position Relative Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-position-relative-ex1.png)
+
+In this example, the blue box is moved 50 pixels down and 100 pixels to the right from its original position within the container.
+## Example 2:Advanced Relative Positioning
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:70]:
```suggestion

## Example 2
```

File: content/css/concepts/position/terms/relative/relative.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+1. Z-index usage to control stacking order of relatively positioned elements.
+2. Percentage-based positioning (50% and 25% left).
+3. Negative values for top positioning to move elements upwards.
+4. How relatively positioned elements interact with each other in the same container.
+
+The green box (Box 3) appears on top due to its higher z-index, despite being declared last in the HTML. The orange box (Box 2) is positioned 50% from the left edge of the container, while the green box is 25% from the left.
+
+``
+
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:124]:
```suggestion
```

File: content/css/concepts/position/terms/relative/relative.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+![z-index and percentage-based positioning.](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-position-relative-ex2.png)
+
+This advanced example demonstrates:
+
+1. Z-index usage to control stacking order of relatively positioned elements.
+2. Percentage-based positioning (50% and 25% left).
+3. Negative values for top positioning to move elements upwards.
+4. How relatively positioned elements interact with each other in the same container.
+
+The green box (Box 3) appears on top due to its higher z-index, despite being declared last in the HTML. The orange box (Box 2) is positioned 50% from the left edge of the container, while the green box is 25% from the left.
```
Comment (mamtawardhani) [inline content/css/concepts/position/terms/relative/relative.md:121]:
```suggestion
The green box (Box 3) appears on top due to its higher` z-index`, even though it is declared last in the HTML. The orange box (Box 2) is positioned 50% from the left edge of the container, while the green box is 25% from the left.
```

**Comment by ozearkhan** [comment]
@mamtawardhani I have commits the changes you suggested. Do i need to do anything else?

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @ozearkhan 😄 

The entry looks good for a second round of review! 🚀

**Comment by ozearkhan** [comment]
Ive noticed some test cases failed in my PR. should i work on fixing them.

**Comment by Sriparno08** [comment]
Hey @ozearkhan, don't worry about that. The reviewers will take care of them.

**Review by Sriparno08 — APPROVED** [review]
Looks good, @ozearkhan!

**Pull request merged**